web/components/answers/answers-panel.tsx

```diff
@@ -128,7 +133,12 @@ export function AnswersPanel(props: {
           )}
         >
           {answerItems.map((item) => (
-            <OpenAnswer key={item.id} answer={item} contract={contract} />
```

Comment:
I kinda like `gap-1` in line 131 now, wdyt
<img width="863" alt="image" src="https://user-images.githubusercontent.com/856034/193896955-00c4b98e-1d4d-4169-9640-ecb460e65423.png">